### DISCUSSION TITLE
Reporting a phishing attempt

### DISCUSSION THREAD

**Discussion by dimonomid** [community/community#21971] [New to GitHub]
<p>Hey guys, I just got a phishing email which looked too legit at first, so let me share it here.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/9/89f99053aa75d5e7c6d6c617158855998a6cba0a.png" data-download-href="/uploads/short-url/jGA6hGYRj1cuAKPkF5akSzWCTQ6.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1329932/181067965-85b5c2df-dca9-4622-99af-6a28b369d03a.png" alt="image" data-base62-sha1="jGA6hGYRj1cuAKPkF5akSzWCTQ6" width="624" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/9/89f99053aa75d5e7c6d6c617158855998a6cba0a_2_624x500.png, https://aws1.discourse-cdn.com/github/original/3X/8/9/89f99053aa75d5e7c6d6c617158855998a6cba0a.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/9/89f99053aa75d5e7c6d6c617158855998a6cba0a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/9/89f99053aa75d5e7c6d6c617158855998a6cba0a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">924×740 81.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>The email comes from the malicious github.coml domain.</p>
<p>The URLs to github.blog and <a href="http://docs.github.com" rel="noopener nofollow ugc">docs.github.com</a> are real (however the actual link first goes to a malicious website), Github indeed had some change re: auth token formats, so phishers are taking advantage of it and trying to make ppl believe they need to regenerate the tokens and login on their phishing site https:// w w w - g t i h u b . c o m, which is a very well functioning copy (or rather proxy maybe) of the actual github. Searching, navigating the repos - all works fine.</p>
<p>I mean, it’s actually one of the most carefully crafted phishing attempts that I’ve personally seen.</p>
<p>Stay safe.</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Thanks <a class="mention" href="/dimonomid">@dimonomid</a> for giving members a heads up. If you are experiencing behavior against our Terms of Service or need to report GitHub account-related abuse, you can reach out to our private support team. We are unable to assist with or discuss personal account details here in the forum.</p>
<p>Please take a look at this link for more details: <a href="https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam" class="inline-onebox">Reporting abuse or spam - GitHub Docs</a></p>
